docs/reference/commandline/attach.md

```diff
@@ -14,6 +14,7 @@ parent = "smn_cli"
 
     Attach to a running container
 
+      --detach-keys       Specify the escape key sequence used to detach a container
```

Comment:
@moxiegirl ah ok my comment wasn't as good as it could then ; @thaJeztah is right, it can be either letter or `ctrl-…` where `…` is a letter. :stuck_out_tongue_closed_eyes: